### ISSUE TITLE
It's there any idea about dual app function

### ISSUE THREAD

**Issue by meixiaofei** [Redmi-K30-5G/device_xiaomi_picasso#5]
![image](https://user-images.githubusercontent.com/20218180/102455786-23c1f180-407b-11eb-93f9-8b282e5f514b.png)
 
I have two question want to ask: 
1. Just like the above pic show.
It's a function build in original MIUI, so cool that can run two same app in the same time,
but i can not find a similar func in any third part build, have any idea about this func?

2. There seems a bug in the latest release
hint VPN components is missing while use some apps

And thanks a lot for u, provide us such a pretty OS, have a really good experience

**Comment by masemoel** [comment]
Yes, I deleted some stuff from the source related to VPN that initially thought it was useless, but I realized later that it isn't, so next version will have VPN working again (very soon).

About dual app function, I haven't searched it in Q. I guess that option can be kinda hidden, as I could do that on Pie with my previous phone (with a shortcut in the navbar buttons) so there should be a way in Q to split the screen as well.
